app/src/org/commcare/views/media/AudioPlaybackButton.java

```diff
@@ -260,4 +294,36 @@ private void pausePlaying() {
         currentState = MediaState.Paused;
         refreshAppearance();
     }
+
+    private void animateProgress(int milliPosition, int milliDuration) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
+            ProgressBar progressBar = (ProgressBar)findViewById(R.id.circular_progress_bar);
+            int startPosition = 0;
+            int progressDuration = 500;
```

Comment:
Sorry I should have been more explicit with my question here, but what I'm confused by is why the progressDuration is always 500?